src/components/OnyxProvider.js

```diff
@@ -22,7 +21,6 @@ const propTypes = {
 const OnyxProvider = props => (
     <ComposeProviders
         components={[
-            NetworkProvider,
```

Comment:
@marcaaron I couldn't quite wrap my head around the purpose of these Provider objects. Would there be any negative side-effects from removing this `NetworkProvider` even after we changed all `withNetwork` references to the new one?

Reply:
This feels like a good one to discuss in Slack.